### DISCUSSION TITLE
Offset exact meaning

### DISCUSSION THREAD

**Discussion by guiferra** [WinMerge/winmerge#1909] [Q&A] [answered]
Good morning,

Can anybody please tell me what what "Offset" exactly means in bottom of WinMerge compare window?
Offset of what and referred to what? (Pls. see image enclosed)
![Capture](https://github.com/WinMerge/winmerge/assets/59984058/15ecd1e4-20e5-4b8e-a0b4-bb10cd9f0df8)

Context:
I am comparing 3 files that should be identical but they are not. For the first 2 files i can see Offset0 and for the third one Offset 427; and I would like to understand what it means.

I check the online documentation but I did not find anything about it.

Thanks for the attention
G

**Comment by sdottaka** [comment, marked as answer]
The offset represents the current caret position. Moving the caret to the beginning displays 0, moving it one byte to the right displays 1.

**Reply by guiferra** [reply]
Sure Takashi! Many thanks for your answer, I was certain it would have been an easy matter, but was not able to see that before.
By the way, it would be also useful to know the meaning of the numbers after letters B, W, L  (always at the bottom row). They change as I move the cursor, but I can not realize what they represent.
Thanks very much for your attention.

**Reply by sdottaka** [reply]
B, W, and L stand for Byte=1byte, Word=2bytes, and Long=4bytes, respectively. For example, the value displayed in L: is 4 bytes of data converted to decimal from the current caret position. Similarly, W: displays 2 bytes of decimal data, and B: displays 1 byte of decimal data.

**Reply by guiferra** [reply]
Ok, got it. Many thanks for replying.
